### PR TITLE
Merge AePPL with history

### PR DESCRIPTION
**Pull request by Armavica**
This PR reverts the AePPL copy/paste and merges AePPL along with its full history.

### CODE REVIEW

File: .github/workflows/tests.yml [hunk 1]
```diff
@@ -90,6 +90,7 @@ jobs:
             pymc/tests/logprob/test_censoring.py
             pymc/tests/logprob/test_composite_logprob.py
             pymc/tests/logprob/test_cumsum.py
+            pymc/tests/logprob/test_joint_logprob.py
```
Comment (ricardoV94) [inline .github/workflows/tests.yml:93]: Oops :)

**Review by ricardoV94 — COMMENTED** [review]
LGTM, what about the license name?

**Comment by Armavica** [comment]
I missed the license name file change, let me see if I can fix that

**Comment by twiecki** [comment]
This is awesome @Armavica, thanks for figuring that out!

**Review by junpenglao — APPROVED** [review]

**Review by twiecki — DISMISSED** [review]
Seems like the commits are here, but none of the code ;).

**Comment by ricardoV94** [comment]
> Seems like the commits are here, but none of the code ;).

The code was already here, now it's just woth history. That's why you don't see any "files changed"

**Review by ricardoV94 — APPROVED** [review]

**Comment by ricardoV94** [comment]
Are the conflicts with main avoidable so we can merge with rebase? Or should we allow a merge commit just this time?

**Comment by twiecki** [comment]
I think a merge commit might be cleaner here.

**Pull request merged**